Gordon360/ApiControllers/ProfilesController.cs

```diff
@@ -803,6 +803,39 @@ public IHttpActionResult UpdateLink(string type, CUSTOM_PROFILE path)
             return Ok();
 
         }
+
+        /// <summary>
+        /// Update mobile phone number
+        /// </summary>
+        /// <param name="value">phoneNumber</param>
+        /// <returns></returns>
+        [HttpPut]
+        [Route("mobile_phone_number/{value}")]
```

Comment:
In other places where the route name had multiple words we've used dashes, I think. I don't know that it matters if we follow that convention though.

Reply:
Is there a description of the new route in the Readme?

Reply:
> Is there a description of the new route in the Readme?

That's a good point. I will work on it now.

Reply:
> In other places where the route name had multiple words we've used dashes, I think. I don't know that it matters if we follow that convention though.

On ProfilesController.cs, all of the route names which have multiple words use '_' instead of dashes.

Ex. image_privacy, mobile_privacy.....